Annex60/Fluid/Movers/BaseClasses/FlowControlledMachine.mo

```diff
@@ -1,6 +1,10 @@
 within Annex60.Fluid.Movers.BaseClasses;
-partial model PrescribedFlowMachine
+partial model FlowControlledMachine
```

Comment:
Confusing names! `ControlledFlowMachine` and `FlowControlledMachine` are very similar names. Why not using the same names as the actual pumps: `FlowControlledMachine` and 'SpeedControlledMachine` instead?